### ISSUE TITLE
Overlaping issue

### ISSUE THREAD

**Issue by iosBeappTeam** [richardtop/CalendarKit#113]
Hello,
First of all, thanks for the work you did. 

The second point is that, i v got some issue about the overlapping, i think it need to be improved 
![image-1](https://user-images.githubusercontent.com/17319330/32849470-821ee7bc-ca2f-11e7-95bd-33c69a6e0e29.jpg)

as you can see on that image the 2 right bottom event should be on the left near the first event.
Do you have any tricks to resolve it please ?

**Comment by richardtop** [comment]
Hi, it's a bug in the library, the way it handles lots of events in the same time interval.
However, even with this bug fixed, in my opinion, there is too many events displayed.
It works best when no more than 2 or 3 events are overlapping.

**Comment by iosBeappTeam** [comment]
Yes i totally agree with you there is too much event displayed, but in mind i dont want to hide some event only because there is too many :/ 

Ok i will try to resolve that, and share the snippet on that issue. 
Feel free if you have any advice

**Comment by richardtop** [comment]
Regarding too many events - then it's just a problem in data representation, and I'd recommend looking into different view, e.g. TableView instead of a Calendar.

If you would like to fix this issue, please take a look at the [TimelineView](https://github.com/richardtop/CalendarKit/blob/master/Source/Timeline/TimelineView.swift) layout code.
